pytrip/cube.py

```diff
@@ -462,16 +463,23 @@ def create_dicom_base(self):
         ds.SOPInstanceUID = '1.2.3'  # !!!!!!!!!!
         ds.StudyInstanceUID = '1.2.3'  # !!!!!!!!!!
         ds.FrameofReferenceUID = '1.2.3'  # !!!!!!!!!
-        ds.StudyDate = '19010101'  # !!!!!!!
-        ds.StudyTime = '000000'  # !!!!!!!!!!
+        ds.SeriesInstanceUID = UID.generate_uid(prefix="2.25.")
+        ds.StudyDate = datetime.datetime.today().strftime('%Y%m%d')
+        ds.StudyTime = datetime.datetime.today().strftime('%H%M%S')
         ds.PhotometricInterpretation = 'MONOCHROME2'
         ds.SamplesPerPixel = 1
         ds.ImageOrientationPatient = ['1', '0', '0', '0', '1', '0']
         ds.Rows = self.dimx
         ds.Columns = self.dimy
         ds.SliceThickness = str(self.slice_distance)
-
         ds.PixelSpacing = [self.pixel_size, self.pixel_size]
+
+        # Add eclipse friendly IDs
```

Comment:
Very interesting ! Can you reproduce what I have below ?
It would be good to know if these problems are related to pydicom version.
If yes, then we should stick to working version in `requirements.txt` file in pytrip

```
→ pip freeze | grep pydicom                                                   ~
pydicom==0.9.9

# grzanka at nz61-6 in ~ [20:07:45]
→ python                                                                      ~
Python 3.6.0 (default, Jan 16 2017, 12:12:55) 
[GCC 6.3.1 20170109] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from dicom.dataset import FileDataset, Tag
>>> ds1 = FileDataset("file", {}, preamble=b"\0" * 128)
>>> ds1.ReferringPhysiciansName = ''
>>> ds1
(0008, 0090) Referring Physician's Name          PN: ''
>>> ds2 = FileDataset("file", {}, preamble=b"\0" * 128)
>>> ds2.add_new(Tag(0x0008,0x0090), 'PN', '')
>>> ds2
(0008, 0090) Referring Physician's Name          PN: ''
```